frontend/src/pages/CoursesList/CoursesList.tsx

```diff
@@ -1,10 +1,65 @@
 import React from 'react';
 
 import { AuthorizedLayout } from 'components/Layout';
+import { PageContainer, CourseButton, CourseActions } from './styled';
```

Comment:
Please regulate imports and divide into 3 groups

Reply:
Fixed lines